src/tasks/package.js

```diff
@@ -40,16 +43,23 @@ const devPackages = [
 ];
 
 module.exports = (config) => {
+  const { name } = meta;
+  const github = gitUsername();
+  const packageName = path.basename(process.cwd());
+  const repository = `${github}/${packageName}`;
+
+  const file = json('package.json');
+  const existingPackage = file.get();
```

Comment:
nitpick 🐦  either `package` or `exitsing` instead of `existingPackage` or an `{Object}` to group the `package.json` related stuff together

```js
const package = {
   name: path.basename(process.cwd()),
   existing: file.get(),
   ...,
} 
```